README.md

```diff
@@ -1,741 +1,16 @@
 # Dynatrace Monitoring as Code
 
-This tool automates deployment of Dynatrace Monitoring Configuration to one or multiple Dynatrace environments.
+Documentation is available on the [monaco documentation website](https://dynatrace-oss.github.io/dynatrace-monitoring-as-code/):
```

Comment:
Personal preference (feel free to ignore): 
Maybe use `[here]` instead of `[monaco documentation website]`.